tests/unittest/_torch/test_executor_request_queue.py

```
@@ -316,141 +316,380 @@ def test_clear_canceled_req_ids(executor_queue):
+        attention_dp_queue):
+    req1 = RequestQueueItem(
+        1,
+        create_mock_request_with_py_schedule_params(attention_dp_rank=0,
+                                                    attention_dp_relax=True))
+
+    new_requests = [req1]
+
+    with patch.object(attention_dp_queue,
+                      '_balance_requests_across_ranks') as mock_balance:
+        mock_balance.return_value = [req1]
+        result = attention_dp_queue._schedule_attention_dp_requests(
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)